config/config.go

```diff
@@ -135,6 +140,11 @@ func (cfg *Configuration) validate(v *viper.Viper) []error {
 	if cfg.AccountDefaults.Events.Enabled {
 		glog.Warning(`account_defaults.events will currently not do anything as the feature is still under development. Please follow https://github.com/prebid/prebid-server/issues/1725 for more updates`)
 	}
+
+	if cfg.PriceFloors.Enabled {
+		glog.Warning(`cfg.PriceFloors.Enabled  will currently not do anything as price floors feature is still under development.`)
+	}
+
 	errs = cfg.Experiment.validate(errs)
 	errs = cfg.BidderInfos.validate(errs)
 	return errs
```

Comment:
Can you also add a table-based unit test to `config/config_test.go` that will test this new `validate` function I'm asking for in the above comment? We can then expand it later when we add the fetch config later on. You could either call `func (pf *AccountPriceFloors) validate(errs []error) []error {` directly or call the top-level validate function to achieve coverage.

Reply:
added test cases